core/src/main/java/org/fao/geonet/resources/CMISConfiguration.java

```diff
@@ -575,6 +578,35 @@ public void init() {
                     Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context to not include relationships.");
                     client.getDefaultContext().setIncludeRelationships(IncludeRelationships.NONE);
                 }
+
+                // Setup default filter. Only include properties that are used by the application.
+                // Having too many may cause performance issues on some systems.
+                // The default is generally an empty string meaning all properties are used.
+                if (StringUtils.isEmpty(client.getDefaultContext().getFilterString())) {
+                    Log.debug(Geonet.RESOURCES, "Changing default CMIS operational context filter.");
+                    // excluding "cmis:secondaryObjectTypeIds" from the list as it could decrease performance on some systems.
+                    client.getDefaultContext().setFilter(new HashSet<>(Arrays.asList(
+                        "cmis:objectTypeId",
```

Comment:
I would replace these values with the constants:

https://chemistry.apache.org/java/javadoc/org/apache/chemistry/opencmis/commons/PropertyIds.html

Reply:
@josegar74, I have changed them to constants